mod-kb-ebsco-java/src/main/resources/domain/mod-kb-ebsco-java/features/titles.feature

```diff
@@ -4,67 +4,110 @@ Feature: Titles
+    * def samplesPath = 'classpath:domain/mod-kb-ebsco-java/features/samples/title/'
 
-  @Undefined
-  Scenario: GET all Titles filtered by tags with 200 on success
-    * print 'undefined'
+    * callonce read('classpath:domain/mod-kb-ebsco-java/features/setup/setup.feature@SetupCredentials')
+    * def package = callonce read('classpath:domain/mod-kb-ebsco-java/features/setup/setup.feature@SetupPackage')
+    * def packageId = package.response.data.id
 
-  @Undefined
-  Scenario: GET all Titles filtered by access-type with 200 on success
-    * print 'undefined'
+#   ================= positive test cases =================
 
-  @Undefined
-  Scenario: GET all Titles filtered by selected with 200 on success
-    * print 'undefined'
-
-  @Undefined
-  Scenario: GET all Titles filtered by type with 200 on success
-    * print 'undefined'
-
-  @Undefined
   Scenario: GET all Titles filtered by name with 200 on success
-    * print 'undefined'
-
-  @Undefined
-  Scenario: GET all Titles filtered by isxn with 200 on success
-    * print 'undefined'
-
-  @Undefined
-  Scenario: GET all Titles filtered by subject with 200 on success
-    * print 'undefined'
+    Given path '/eholdings/titles'
+    And param filter[name] = 'Test'
+    When method GET
+    Then status 200
+    And match responseType == 'json'
 
-  @Undefined
-  Scenario: GET all Titles filtered by publisher with 200 on success
-    * print 'undefined'
+  Scenario: POST Titles should create a new Custom Title with 200 on success
+    Given path '/eholdings/titles'
+    And def titleName = random_string()
+    And def requestEntity = read(samplesPath + 'createTitle.json')
+    And param filter[publisher] = requestEntity.data.attributes.publisherName
+    When method GET
+    Then status 200
+    And def initial_num_records = response.meta.totalResults
+
+    Given path '/eholdings/titles'
+    And request requestEntity
+    When method POST
+    Then status 200
+    And def titleId = response.data.id
+
+    #waiting for title creation
+    * eval sleep(20000)
+
+    Given path '/eholdings/titles'
+    And param filter[publisher] = requestEntity.data.attributes.publisherName
+    When method GET
+    Then status 200
+    And match response.meta.totalResults == initial_num_records + 1
+
+    Given path '/eholdings/titles', titleId
+    When method GET
+    Then status 200
+    And def attributes = response.data.attributes;
+    And match attributes.name == requestEntity.data.attributes.name
+    And match attributes.publisherName == requestEntity.data.attributes.publisherName
+    And match attributes.publicationType == requestEntity.data.attributes.publicationType
 
-  @Undefined
-  Scenario: GET all Titles should return 400 if filter param is missing
-    * print 'undefined'
+  Scenario: GET Title by id with 200 on success
+    Given path '/eholdings/titles/297646'
```

Comment:
you may use a constant and define it after the declaration of title name https://github.com/folio-org/folio-integration-tests/pull/286/files#diff-e407eaa6007bc71d21d9b098302af8fd3bfb5046110b66a5964e6cca211d1cd5R24 and than change the request to  Given path '/eholdings/titles', sampleTitleId